skimage/measure/tests/test_polygon.py

```diff
@@ -11,18 +11,30 @@
 
 
 def test_approximate_polygon():
```

Comment:
I would split this test in several ones, with a name that reflects the idea behind the each test.